### PR TITLE
Adjusts the readability on the gasharpoon

### PR DESCRIPTION
**Pull request by cowbot92**
# Document the changes in your pull request

Adjusts the readability to be more in-line with what I intended it to be originally, while still being visually pleasing.

# Why is this good for the game?

readability good.

# Testing

text change

# Wiki Documentation

Adjust it all to GasHarpoon

# Changelog

:cl:  

spellcheck: Gas-Harpoon -> GasHarpoon

/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/busterarm/gasharpoon.dm [hunk 1]
```diff
@@ -1,13 +1,13 @@
 /obj/item/clothing/gloves/gasharpoon
-	name = "gas-harpoon"
+	name = "GasHarpoon"
```
Comment (AMyriad) [inline code/game/objects/items/devices/busterarm/gasharpoon.dm]: we don't capitalize these kinds of items in our code

**Review by AMyriad — COMMENTED** [review]
this is just a revert, man

File: code/game/objects/items/devices/busterarm/gasharpoon.dm (hunk 1 above)
Comment (cowbot92) [reply code/game/objects/items/devices/busterarm/gasharpoon.dm]: ok

**Comment by cowbot92** [comment]
> this is just a revert, man

This is the way it should be.

**Comment by Firewolf34** [comment]
So I get why you want the name to not be hyphenated, and that makes sense given the reference, but the new description was nice... Gotta reel in those space carps after all!

**Review by Chubbygummibear — APPROVED** [review]
LooksGoodToMe

**Pull request merged**